replica/extents.go

```diff
@@ -62,3 +66,47 @@ func (u *UsedGenerator) findExtents(c chan<- int64) {
 		}
 	}
 }
+
+func createTempFile(path string) error {
```

Comment:
If the file is empty but fs wasn't supported, will `Fiemap` return error? If so, you don't need to create another file. Just open `head` disk in RO and test it. https://github.com/rancher/longhorn-engine/blob/master/replica/replica.go#L177

Reply:
`Fiemap` also work on empty files, it just requires `size` to generate extents.